geode-core/src/test/java/org/apache/geode/internal/cache/functions/TestFunction.java

```diff
@@ -177,6 +178,8 @@ public void execute(FunctionContext context) {
       executeFunctionBucketFilter(context);
     } else if (id.equals(TEST_FUNCTION_NONHA_NOP)) {
       execute1(context);
+    } else if (id.equals(TEST_FUNCTION_FIREANDFORGET_ONALL_SERVERS)) {
```

Comment:
I would suggest just creating a new function for your test. I think this TestFunction class needs to go away, it's crazy to have this one big object that has a bunch of unrelated methods for different tests.